app/decorators/place_decorator.rb

```diff
@@ -0,0 +1,3 @@
+class PlaceDecorator < Draper::Decorator
+  delegate_all
```

Comment:
why this is empty? are you using it somewhere?

Reply:
i've create that file but i dont need it - imo it's not a problem to keep that in repository because probably i'll need this in next 2 or 3 weeks :)

Reply:
sorry dude, we've got a convetnion to create files and classes only if you're using it

Reply:
but in that case i think it's ok